neo4j/v1/session.py

```diff
@@ -103,11 +103,13 @@ class Driver(object):
     def __init__(self, url, **config):
         self.url = url
         parsed = urlparse(self.url)
-        if parsed.scheme == "bolt":
+        transports = ['bolt']
+        if parsed.scheme in transports:
             self.host = parsed.hostname
             self.port = parsed.port
         else:
-            raise ValueError("Unsupported URL scheme: %s" % parsed.scheme)
+            raise ProtocolError("Unsupported transport: '%s' in url: '%s'. Supported transports are: '%s'." %
+                                (parsed.scheme, url, transports))
```

Comment:
Sure. These changes were made to fit how the Java Driver did. I have no preference here.